docs/UserGuide.md

```diff
@@ -104,31 +103,33 @@ Examples:
   Selects the 1st person in the results of the `find` command.
 
 #### Clearing all entries : `clear`
-Clears all entries from the address book.<br>
-Format: `clear`  
+Clears all entries from the task list.<br>
+Format: `clear` 
 
 #### Exiting the program : `exit`
 Exits the program.<br>
 Format: `exit`  
 
 #### Saving the data 
-Address book data are saved in the hard disk automatically after any command that changes the data.<br>
+Task List data are saved in the hard disk automatically after any command that changes the data.<br>
 There is no need to save manually.
 
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with 
-       the file that contains the data of your previous Address Book folder.
+       the file that contains the data of your previous Tasks folder.
        
 ## Command Summary
 
 Command | Format  
 -------- | :-------- 
-Add | `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Add | `add NAME d/duedate l/location [t/TAG]...`
+Edit | `edit INDEX [Detail Type] x/detail change` where x is either d or l
 Clear | `clear`
 Delete | `delete INDEX`
 Find | `find KEYWORD [MORE_KEYWORDS]`
 List | `list`
 Help | `help`
 Select | `select INDEX`
+Exit | `exit`
```

Comment:
The 'Command summary' section will be used heavily by our testers. Try to make it easy to use as possible. e.g. give examples for each command.